frontend/public/style/_vars.scss

```diff
@@ -78,6 +78,7 @@ $color-secret-dark: $pf-color-orange-400;
 $color-service-dark: $pf-color-light-green-500;
 $color-text-muted: $color-pf-black-600;
 $color-text-secondary: $color-pf-black-600;
+$color-vmi-dark: #002F5D;
```

Comment:
this can be PF variable `--pf-chart-color-blue-500`

Reply:
I'd disagree @rawagner we should only use global variables here unless we have a specific one for the badge. We can't say for sure that in the future the chart color could change and then we would inherit it which is not ideal.
I know there is a plan to make PF color variables more available but for now either doing it this way,  or creating a KV variable is probably the best approach.